Makefile

```diff
@@ -171,13 +171,25 @@ dist-image-push: dist-image  $(subst image_,push_image_,$(ALL_SUPPORTED_IMG_OS_A
 	@for img in $(ALL_IMAGE_REFS); do TARGETOS=$$(echo $$img | sed -e 's/.*:[^-]\+-\([^-]\+\).*/\1/'); TARGETARCH=$${img/*-}; podman manifest add $(IMAGE_REPO):$(IMAGE_TAG) $(PODMAN_OPTS) $$img --os $${TARGETOS} --arch $${TARGETARCH}; done
 	@podman manifest push $(IMAGE_REPO):$(IMAGE_TAG) $(IMAGE_REPO):$(IMAGE_TAG)
 
+.PHONY: dev
+dev: IMAGE_REPO=localhost:5000/ec
+dev: PODMAN_OPTS=--tls-verify=false
```

Comment:
I'm a perpetual Makefile noob. What does it mean when the target is repeated?

Reply:
It's just a [weird syntax](https://www.gnu.org/software/make/manual/make.html#Target_002dspecific) to set the variables per-target. It makes for better UX for the `dev` use case to have these preset

Reply:
Oh, and I think if one don't have 50cm long gray beard you're not legally allowed to write Makefiles, so we're all Makefile noobs at this point.